index.html

```diff
@@ -8,34 +8,42 @@
     </head>
     <body>
         <header class="header">
-            <span class="logo-link">
+            <div class="logo-link">
                 <a href=".//index.html"><img src=".//img/logo-black.png" alt="Логотип надпись device черным цветом"></a>
-            </span>
+            </div>
             <div class="header-buttons">                
-                <div class="search">                    
+                <div class="search-container">                    
                     <label class="visually-hidden" for="input-search">Поиск по сайту</label>
                     <input class="input-search" type="text" name="search" id="input-search" placeholder="Поиск по сайту">                                    
-                    <span class="button-search"><a href="#">Найти</a></span>
+                    <div class="button-search"><a href="#">Найти</a></div>
                 </div>
                 <div class="user-buttons">
                     <ul class="user-registration">
-                        <li class="button-registration"><a href=".//registration.html">Войти</a></li>
+                        <li class="button-registration" tabindex="-1"><a href=".//registration.html">Войти</a></li>
                     </ul>
                     <ul class="user-purchase">
-                        <li class="button-compare"><a href="#">Сравнить</a></li>
+                        <li class="button-compare" tabindex="-1"><a href="#">Сравнить</a></li>
```

Comment:
why tabindex -1, what for do you remove this link from navigation flow?

Reply:
Removed it. It was after lecture of accessibility and i checked how it works and haven't removed